src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -25,12 +26,14 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
+    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
```

Comment:
Actually, might be best to just keep it as default. Some fights you end up holding 2 charges because needed high ST healing comes in at a rate faster than 45s (I think M G'dan soulsever was an example of this) so you want as many charges going into it as possible. Other times you just kinda use it flimsily whenever. It's good to use it when it's available, so knowing cast efficiency is valuable, but sometimes you end up changing your usage such that it would affect cast efficiency based on fights.